pelican/plugins/image_process/image_process.py

```diff
@@ -703,17 +681,34 @@ def process_picture(soup, img, group, settings, derivative):
             img.insert_before(s["element"])
 
 
+def try_open_image(path):
+    try:
+        i = Image.open(path)
+    except UnidentifiedImageError:
+        logger.warning(
+            "%s Source image %s is not supported by Pillow.",
```

Comment:
When putting a filename like this in a log message, I like to wrap the name in quotes (single or double).

This is probably more for Windows, as Rich (the log processing library) doesn't consistently pick out Windows filenames.